pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -332,6 +332,10 @@ public static class QueryOptionValue {
         public static final String ROUTING_FORCE_HLC = "FORCE_HLC";
         public static final String DEFAULT_IN_PREDICATE_SORT_THRESHOLD = "1000";
       }
+
+      public static class SharedValueKey {
+        public static final String LOCAL_JOIN_RIGHT_TABLE_PREFIX = "localJoinRightTable_";
```

Comment:
I don't think I am able to follow this concept. Can you please elaborate or explain with some brief comments ?